mapbox_tilesets/scripts/cli.py

```diff
@@ -384,7 +384,7 @@ def delete_source(username, id, force, token=None):
         mapbox_api, username, id, mapbox_token
     )
     r = requests.delete(url)
-    if r.status_code == 201:
+    if r.status_code == 204:
```

Comment:
where do you get this 204 from?

Reply:
We updated the HTTP status code for delete action this from 201 to 204. This was due to the fact that `201` is for created while `204` is no content.

Reply:
https://github.com/mapbox/api-tilesets/blob/bdd26c2f4be7c972831b14aec297146aae20fede/routes/tilesets-source.js#L98

Reply:
found it. thnx. neat